packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/go/projectstructure.md

```diff
@@ -0,0 +1,20 @@
+There are two main repositories for the embedded SDK and sample applications.
```

Comment:
To summarize, we will merge the "Project structure" and "Download from GitHub" sections into one new section and place this new section where the "Download from GitHub" section is currently located. Totally agree with the restructuring. Since the changes (though not significant) will affect the other tech stacks (ASP.NET, Java, etc.) because the headers are in common files, can we make this change separately and possibly track it with its own Jira ticket?

Reply:
Yup, that makes sense to me. Please create a new JIRA for this, and add it to the "OIE cleanup work, post-Tuscany" epic. Thanks!

Reply:
A jira ticket https://oktainc.atlassian.net/browse/OKTA-419326 has been created.

Reply:
Great, cheers man.